lib/routes/novel/ptwxz.js

```diff
@@ -24,24 +25,57 @@ module.exports = async (ctx) => {
 
     const title = $('span>h1').text();
 
-    const list = $('.grid>tbody>tr>td>li>a');
+    const list = $('.grid>tbody>tr>td>li');
+
+    const chapter_item = list
+        .find('a')
+        .map((_, e) => ({
+            title: e.children[0].data,
+            link: e.attribs.href,
+            // pubDate // no date info found
+        }))
+        .get();
+
+    const items = await asyncPool(3, chapter_item, async (item) => {
+        const cache = await ctx.cache.get(item.link);
```

Comment:
use `ctx.cache.tryGet` instead of explicitly set/get cache.

Reply:
so `ctx.cache.get` -> `ctx.cache.tryGet`, i am not familiar with node/js programming and so I basically just test and copt from biquege.js

Reply:
> so ctx.cache.get -> ctx.cache.tryGet
No